src/tests/pages/orders.page.js

```diff
@@ -0,0 +1,204 @@
+        this.campStudentsNumberLocator = this.page.locator("#camp-students");
+        this.campStudentsAgeLocator = this.page.locator("#camp-age");
+        this.campAdultsNumberLocator = this.page.locator("#camp-adults");
+
+        this.natureStudentsNumberLocator = this.page.locator("#nature-students");
+        this.natureStudentsAgeLocator = this.page.locator("#nature-age");
+        this.natureAdultsNumberLocator = this.page.locator("#nature-adults");
+        this.natureStartTimeLocator = this.page.locator("#nature-start_time");
+        this.natureStartFoodLocator = this.page.locator("#nature-start_food");
+        this.natureEndTimeLocator = this.page.locator("#nature-end_time");
+        this.natureEndFoodLocator = this.page.locator("#nature-end_food");
+
+        this.toastErrorLocator = this.page.locator(".toast-error");
+        this.toastMessage = this.page.locator(".toast-message");
+        this.fieldErrorLocator = this.page.locator(".invalid-feedback");
+        this.orderConfirmationTitle = this.page.getByRole('heading', { name: 'Děkujeme za objednávku' });
+
+        // Group form fields:
+        this.orderFormFields = {
+            icoField: this.icoFieldLocator,
+            clientField: this.clientFieldLocator,
+            addressField: this.addressFieldLocator,
+            substituteField: this.substituteFieldLocator,
+            contactNameField: this.contactNameFieldLocator,
+            contactPhoneField: this.contactPhoneFieldLocator,
+            contactEmailField: this.contactEmailFieldLocator,
+            startDate1Field: this.startDate1FieldLocator,
+            endDate1Field: this.endDate1FieldLocator,
+            startDate2Field: this.startDate2FieldLocator,
+            endDate2Field: this.endDate2FieldLocator,
+            startDate3Field: this.startDate3FieldLocator,
+            endDate3Field: this.endDate3FieldLocator
+        };
+
+        // Group camp information form fields:
+        this.campOrderFields = {
+            campDatePartField: this.campDatePartLocator,
+            campStudentsNumberField: this.campStudentsNumberLocator,
+            campStudentsAgeField: this.campStudentsAgeLocator,
+            campAdultsNumberField: this.campAdultsNumberLocator
+        };
+
+        // Group school in nature information form fields:
+        this.natureOrderFields = {
+            natureStudentsNumberField: this.natureStudentsNumberLocator,
+            natureStudentsAgeField: this.natureStudentsAgeLocator,
+            natureAdultsNumberField: this.natureAdultsNumberLocator,
+            natureStartTimeField: this.natureStartTimeLocator,
+            natureStartFoodField: this.natureStartFoodLocator,
+            natureEndTimeField: this.natureEndTimeLocator,
+            natureEndFoodField: this.natureEndFoodLocator
+        };
+    }
+
+    async open() {
+        await this.page.goto("/objednavka/pridat");
+    }
+
+    async fillClientInfo(
+        ico = icoNumber,
```

Comment:
Moc se mi nelíbí tahat ty samé fixtures jak do testu tak do page objektu (tím spíš, že se nikde ty defaultní hodnoty z page objektu nepoužívají, na všech místech použití se vyplňují znovu).
```suggestion
        ico,
```